### PR TITLE
Feature table

### PR DESCRIPTION
**Pull request by CalebBassham**
Added a table component and demo

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,6 +7,12 @@ import badge from './components/badge';
 import alert from './components/alert';
 import article from './components/article';
 import pre from './components/pre';
+<<<<<<< HEAD
```
Comment (TotomInc) [inline src/index.js]: Uh you forgot to resolve the conflicts, but I'll do it. 😉

**Review by TotomInc — CHANGES_REQUESTED** [review]

**Comment by TotomInc** [comment]
@DarkPaladinUHC can you resolve the conflicts or let me an access to push on your PR by doing [this](https://i.stack.imgur.com/wu4Wd.png)?

EDIT: it would be better to let me an access since I've also fixed a dependency mismatch.

**Comment by CalebBassham** [comment]
I allow maintainer edits with all my pull requests.

**Comment by CalebBassham** [comment]
I thought that I fixed that conflict but I guess not. \\\_O\_/

**Review by TotomInc — APPROVED** [review]

**Pull request merged**